nautobot/core/management/commands/generate_ui.py

```diff
@@ -1,3 +1,4 @@
+# TODO: is this command still needed and relevant?
```

Comment:
This was implementing a concept of cutting everything over to the new ui by putting the javascript UI in `base.html` and loading the `base.html` `{% block content %}` in a hidden div to load asynchronously from the DOM by javascript. @bryanculver did we want to keep this concept around?

Reply:
I'm replacing this command with `build`. So I'll just remove it in my branch.